lnd.go

```diff
@@ -180,14 +180,41 @@ func Main(cfg *Config, lisCfg ListenerCfg, implCfg *ImplementationCfg,
 		network,
 	)
 
+	ctx := context.Background()
+	ctx, cancel := context.WithCancel(ctx)
+	defer cancel()
+
 	// Enable http profiling server if requested.
 	if cfg.Profile != "" {
+		// Create the http handler.
+		profileRedirect := http.RedirectHandler("/debug/pprof",
```

Comment:
nit: pre-existing, but would still be nice to fix the formatting, if we're touching the code anyway.

Reply:
def!